docs/plugins.md

```diff
@@ -44,6 +44,17 @@ By default, the hyperlink on the current page is recognized and the content is s
       // To avoid search index collision
       // between multiple websites under the same domain
       namespace: 'website-1',
+
+      // Use different indexes for path prefixes (namespaces).
+      // NOTE: Only works in 'auto' mode.
+      //
+      // When initialiazing an index, we look for the first path from the sidebar.
+      // If it matches the prefix from the list, we switch to the corresponding index.
+      pathNamespaces: ['/zh-cn', '/ru-ru', '/ru-ru/v1'],
```

Comment:
Please modify the [index.html](https://github.com/docsifyjs/docsify/blob/develop/index.html) configuration for testing.

Reply:
Done